Xamarin.Forms.Platform.Android/Platform.cs

```diff
@@ -61,6 +63,7 @@ public class Platform : BindableObject, INavigation, IDisposable, IPlatformLayou
 
 		internal Platform(Context context, bool embedded)
 		{
+			PackageName = context.PackageName;
```

Comment:
For our case here I don't think we need this tracked and you can safely pass `null` for the third argument of `GetIdentifier`. This will default to look for resources in the project (and not the framework) which is what the reflection code does as well.

Reply:
@garuma And I assume this all relies on the design time build being run, to copy the image files to the right place? Not that that's a problem, as I believe the previewer always does that, but I wanted to confirm the mechanism.

Reply:
When running the app that doesn't seem to work

```c#
// returns the resource id
context.Resources.GetIdentifier(name, value, context.PackageName);

// this just returns a 0
context.Resources.GetIdentifier(name, value, null);
```

Reply:
Just checked the difference between designer and native Android and you are right, the designer re-implementation (in Google's layoutlib) looks to be more lenient and doesn't require a package argument. The framework version however will bail out if it's not given.

Looking at the latest changes I think your fallback approach makes sense. If there is a NuGet package that's produced from this we can double check it works on Monday.

@BretJohnson yes this will use the resources from the generated build output same as what the Android designer does for rendering.